app/code/Magento/Sales/Model/AdminOrder/Create.php

```diff
@@ -33,6 +33,7 @@ class Create extends \Magento\Framework\DataObject implements \Magento\Checkout\
      */
     const XML_PATH_DEFAULT_EMAIL_DOMAIN = 'customer/create_account/email_domain';
 
+    const XML_PATH_EMAIL_REQUIRED_CREATE_ORDER = 'customer/create_account/email_required_create_order';
```

Comment:
Please, make constant private

Reply:
Judging by the current code not obeying such practice I would suggest that change be made in a separate PR aimed at resolving this in the entire class.

Reply:
Having said that - I notice the PHPDOC is missing for this constant.

Reply:
@driskell we can't make existing constants as private due to backward compatibility, but the new one should be private.